config/routes.rb

```diff
@@ -1,5 +1,16 @@
 Rails.application.routes.draw do
+  get 'sessions/new'
+  get 'sessions/create'
+  get 'sessions/destroy'
+  resources :users
   resource :home, controller: :home, only: [:show]
   root to: 'home#show'
+  get 'signup' => "users#new"
+  post 'signup' => 'users#create'
+  get 'signin' => 'sessions#new'
+  post 'sessions/new' => 'sessions#create'
+  delete 'signout' => 'sessions#delete'
+  get 'attend' => 'events#attend_event'
+  resources :events
```

Comment:
- [ ] please specify the actions used.

Reply:
Sorry, can you explain more on this point? Are you referring to split resources into several routes or something else? Thanks in adavance

Reply:
What about this? 
https://github.com/Israel-Laguan/Private-Events/blob/f35174555e1430ff38557501d8b1003eb6312f86/config/routes.rb#L13